src/index.js

```diff
@@ -1,40 +1,84 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Take a Shower',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Cook today',
-    completed: false,
-    index: 10,
-  },
-  {
-    description: 'Buy groceries',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: ' Homework',
-    completed: false,
-    index: 8,
-  },
-  {
-    description: ' List Project',
-    completed: false,
-    index: 4,
-  },
-];
+/*
+ localStorage.setItem('tasks', JSON.stringify(tasks));
+*/
+
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+// Add an element
+const addTask = document.getElementById('addItem');
+addTask.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    window.location.reload();
+    const maxId = tasks.length;
+    tasks.push({
+      description: addTask.value,
+      Completed: false,
+      index: maxId,
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    window.location.reload();
+  }
+});
 
 const show = () => {
   const getList = document.getElementById('todo-list');
   for (let i = 0; i < tasks.length; i += 1) {
     const item = document.createElement('li');
-    item.innerHTML = `<input type="checkbox" class="check" />  ${tasks[i].description}`;
+    let chek = '';
+    if (tasks[i].Completed === true) {
+      chek = 'checked';
+    }
+    item.innerHTML = `<input  type="checkbox" ${chek} class="check" onchange=checkboxx(${i}) />  ${tasks[i].description}
+    <button onclick=deletebtn(${tasks[i].index}) class="deletebtn">Delete</button>`;
     getList.appendChild(item);
   }
 };
 
```

Comment:
Please add edit functionality to your to-do-list as mentioned on project requirements. ✍️
<img width="698" alt="Screenshot 2023-01-29 at 1 25 11 AM" src="https://user-images.githubusercontent.com/25887253/215293851-9c9036c8-4e40-49f6-b00f-6bf915822952.png">